frontend/awx/resources/inventories/inventoryHostsPage/inventoryHostsGroups.tsx

```diff
@@ -0,0 +1,64 @@
+import { CubesIcon } from '@patternfly/react-icons';
```

Comment:
Change name of file because this view shows the groups of a single host.
```inventoryHostGroups.tsx```

Reply:
done